test/index.spec.js

```diff
@@ -446,8 +446,8 @@ describe('daemons', () => {
 
         expect(daemon).to.have.property('apiAddr')
         expect(daemon).to.have.property('gatewayAddr')
-        expect(daemon.apiAddr).to.be.instanceof(multiaddr)
-        expect(daemon.gatewayAddr).to.be.instanceof(multiaddr)
+        expect(multiaddr.isMultiaddr(daemon.apiAddr)).to.equal(true)
+        expect(multiaddr.isMultiaddr(daemon.gatewayAddr)).to.equal(true)
```

Comment:
This fixes @thisconnect issue reported on https://github.com/ipfs/js-ipfsd-ctl/issues/158

Note for the future. Using `instanceof` in JS land is bad practice, you can never count on getting the right class/prototype at the right type because npm will install multiple versions (thankfully).